borg/helpers.py

```diff
@@ -235,16 +235,21 @@ def parse_timestamp(timestamp):
         return datetime.strptime(timestamp, '%Y-%m-%dT%H:%M:%S').replace(tzinfo=timezone.utc)
 
 
+def load_excludes(fh):
+    """Load and parse exclude patterns from file object. Empty lines and lines starting with '#' are ignored, but
+    whitespace is not stripped.
+    """
+    patterns = [line.rstrip('\r\n') for line in fh if not line.startswith('#')]
```

Comment:
> maybe it could just be .rstrip() which strips all whitespace.

This change is meant to add a test, not to change behaviour.

> also, you could create an iterator here instead of a list.

Will do so in the follow-up change which has to move things around. The code in this change is unchanged from the original.